docs/beyond_basics/101-181-metalad.rst

```diff
@@ -4,10 +4,276 @@ Metadata-Management with MetaLad
+
+TODO: This needs and example
+
+
+Creating your own extractor
+"""""""""""""""""""""""""""
+
+TODO: This also needs an example
+
+
+Distributing and Getting Metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once metadata has been added to a dataset, it can be distributed and retrieved.
+Instead of creating and adding metadata yourself, you could download fitting pre-existing metadata.
+Similarly, instead of repeating a ``meta-add`` process for one and the same files across hierarchies of datasets, metadata added into one dataset can be exported into other datasets.
+Regardless of whether it is a distribution or a retrieval process, though, an export with MetaLad will only concern the *metadata*, and never the primary data.
+
+Download Metadata from a remote repository
+""""""""""""""""""""""""""""""""""""""""""
+
+Let's start by creating a place where someone else's metadata could live.
+
+.. code-block:: bash
+
+    $ datalad create metadata-assimilation
+    $ cd metadata-assimilation
+
+Because MetaLad stores metadata in :term:`Git`'s object store, we use Git to directly fetch metadata from a remote repository, such as this demo on :term:`GitHub`: ``https://github.com/christian-monch/metadata-test.git``.
+Because metadata added by MetaLad is not transported automatically but needs to be specifically requested, the command to retrieve it looks unfamiliar to non-Git-users: It identifies the precise location of :term:`ref` that contains the metadata.
+
+.. find-out-more:: Exactly where is metadata stored?
+   :name: fom-metadataobjecttree
+
+   MetaLad employs an internal metadata model that makes the following properties possible:
+
+   * Metadata has a version encoded, but isn't itself version controlled
+   * Metadata should not be transported if not explicitly requested
+   * It should be possible to only retrieve parts of the overall metadata tree, e.g. certain sub-nodes
+
+   TODO this needs much more
+
+.. code-block:: bash
+
+    $ git fetch \
+      "https://github.com/christian-monch/metadata-test.git" \
+      "refs/datalad/*:refs/datalad/*"
+
+The metadata is now locally available in the Git repository ``metadata-repo``.
+You can verify this by issuing the command ``datalad meta-dump -r``, which will list all metadata in the repository.
+
+
+Publish metadata to a Git-Repository
+""""""""""""""""""""""""""""""""""""
+
+You can also push your metadata to a remote :term:`sibling` (if you have write :term:`permissions`).
+This, too, uses a Git command to push only specific :term:`ref`\s.
+Let's assume you are in the directory that contains the git repository with your metadata, then you can push your metadata to a remote git repository ``<your repository>``::
+
+    $ git push "<your repository>" "refs/datalad/*:refs/datalad/*"
+
+You will notice that no primary data is stored in the repository ``metadata-destination``. That allows you to publish metadata without publishing the primary data at the same time.
+
+
+Querying metadata
```

Comment:
Perhaps this would be a good place to add some note about and links to the catalog? Or a similar section titled something like `Using metadata`

Reply:
@adswa E.g. under a `Using metadata` section (or `Let's put it to use`) we could say something along the lines of:

- now that we know all about metadata and how it is handled by metalad, a final note on its utility
- metadata, especially when originating from different sources and harmonized to a single schema, provide the powerful opportunity to aid data discoverability
- an example of a good use case for metadata is therefore search or browsing interfaces or data bases, such as data portals graph query databases, etc
- Metalad-extracted metadata can be used in workflows to generate such interfaces, an example being datalad catalog
- the example: add metadata to datalad datasets, extract metadata with metalad and multiple extractors, translate extracted metadata to the catalog schema, submit it to datalad-catalog in order to generate catalog entries, which can all be browsed in a user friendly web-based interface.

Reply:
I have added this pretty much exactly in your words - thanks for the excellent suggestion!